docs/vault/upgrading.md

```diff
@@ -1,122 +0,0 @@
-# Upgrading the Vault Client
```

Comment:
This file has been removed but wasn't documented elsewhere. Maybe the content should be moved to the [Operating the Vault Client](https://docs.interlay.io/#/vault/guide?id=operating-the-vault-client) page